iib/workers/tasks/build.py

```diff
@@ -1498,7 +1498,13 @@ def _adjust_operator_bundle(manifests_path, metadata_path, organization=None, pi
     """
     package_name, labels = _apply_package_name_suffix(metadata_path, organization)
 
-    operator_manifest = OperatorManifest.from_directory(manifests_path)
+    try:
+        operator_manifest = OperatorManifest.from_directory(manifests_path)
+    except ruamel.yaml.YAMLError as e:
+        error = f'Not a valid yaml file: {e}'
```

Comment:
Can we be more specific? How about: "The Operator Manifest is not in a valid YAML format: {e}"